src/main/kotlin/com/terraformation/backend/i18n/Gibberish.kt

```diff
@@ -0,0 +1,16 @@
+import java.util.*
+
+/**
+ * Converts an English string to gibberish for localization testing:
+ * 1. Reverse the order of words
+ * 2. Replace each word with the base64 encoding of its UTF-8 representation, minus padding
+ *
+ * Note that this function does not preserve placeholder tokens.
+ */
+fun String.toGibberish(): String {
```

Comment:
TerrawareGenerator runs as part of the build process; at the point it runs, `src/main` hasn't been compiled yet (and can't be, because it depends on the generated code).

I agree it's not ideal to have multiple copies of this logic but sharing a single copy across the build and the application would require adding another build stage which didn't seem worth it for a function this simple.

Reply:
got it, thanks!